client/ayon_core/pipeline/colorspace.py

```diff
@@ -699,6 +699,33 @@ def get_ocio_config_views(config_path):
     )
 
 
+def _get_config_path_from_profile_data(
+        data, data_type, template_data) -> dict:
+    """Get config path from profile data.
+
+    Args:
+        data (dict[str, Any]): Profile data.
+        data_type (str): Profile type.
+        template_data (dict[str, Any]): Template data.
+
+    Returns:
+        dict[str, str]: Config data with path and template.
+    """
+    template = data[data_type]
+    result = StringTemplate.format_strict_template(
+        template, template_data
+    )
+    normalized_path = str(result.normalized())
+    if not os.path.exists(normalized_path):
+        log.warning(f"Path was not found '{normalized_path}'.")
+        return None
```

Comment:
Use the same variable names and don't use `dict` for type hint as it's not supported in Python 3.8
```suggestion
def _get_config_path_from_profile_data(
    profile, profile_type, template_data
):
    """Get config path from profile data.

    Args:
        profile(dict[str, Any]): Profile data.
        profile_type (str): Profile type.
        template_data (dict[str, Any]): Template data.

    Returns:
        dict[str, str]: Config data with path and template.
    """
    template = profile[profile_type]
    result = StringTemplate.format_strict_template(
        template, template_data
    )
    normalized_path = str(result.normalized())
    if not os.path.exists(normalized_path):
        log.warning(f"Path was not found '{normalized_path}'.")
        return None
```